index.bs

```diff
@@ -3612,7 +3608,7 @@ To <dfn>maybe replace event-level report</dfn> given an [=attribution source=]
     If this assumption does not hold then |rateLimitRecord| might be null.
 
 1. [=set/Remove=] |rateLimitRecord| from the [=attribution rate-limit cache=].
-1. Return "<code>[=event-level-report-replacement result/add-new-report=]</code>".
+1. Return null.
```

Comment:
Why are we changing this? The `add-new-report` value is now unused.

Reply:
Can we just remove references to `add-new-report`?

Reply:
It seems odd to me to name a value when there is no handling for it. I don't feel strongly about it. Would you rather have the return, `add-new-report`, when we don't need to handle it explicitly in this sequence?

Reply:
Since we have to return a value from the algorithm in all paths, a named value is more descriptive than `null` and easier to search for. You can even add an assertion after the switch that the value is equal to that value.

Reply:
Done!